src/wallet/rpcdump.cpp

```diff
@@ -1364,20 +1364,13 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
         EnsureWalletIsUnlocked(pwallet);
 
         // Verify all timestamps are present before importing any keys.
-        const Optional<int> tip_height = locked_chain->getHeight();
-        now = tip_height ? locked_chain->getBlockMedianTimePast(*tip_height) : 0;
```

Comment:
1f4b604

If #17443 gets first + `GetLastBlockTime`, you may avoid to call `findBlock` here.